static/js/zamboni/collections.js

```diff
@@ -268,6 +268,8 @@ if (addon_ac.length) {
                 .data('item.autocomplete', item)
                 .append('<a><img src="' + item.icons['32'] + '" alt="">' + _.escape(item.name) + '</a>')
                 .appendTo(ul);
+        } else {
+            return ul;
```

Comment:
Does `_renderItem` ignore the ul returned here - seems weird since the bit in the if block is actually returning the li.